spec/requests/api/v1/announcements/reactions_spec.rb

```diff
@@ -56,4 +54,8 @@
       end
     end
   end
+
+  def escaped_emoji
+    CGI.escape('😂')
```

Comment:
This previously worked fine as a string argument to the controller, because that's all happening at the ruby level. The request spec flows through http stack, and was giving invalid url error when interpolated directly without escaping first.